vowpalwabbit/reductions/csoaa_ldf.cc

```diff
@@ -356,31 +306,29 @@ void predict_csoaa_ldf(ldf& data, single_learner& base, multi_ex& ec_seq_all)
   if (ec_seq_all.empty()) return;  // nothing to do
 
   data.ft_offset = ec_seq_all[0]->ft_offset;
-  // handle label definitions
-  auto ec_seq = process_labels(data, ec_seq_all);
```

Comment:
While I am in favour of this change isn't this technically the removal of a supported feature? That feature being the label dependent features part of csoaa ldf?

Reply:
yeah I think last commit may be mistake - let me revert it.

Reply:
please ignore previous comment.
This part is clearly redundant if we believe that problematic 'l' namespace check can be removed.

Reply:
Are we still proposing changing the namespace to some reserved namespace? Or are we removing LDF entirely?